samples/azure-quantum/hello-world/HW-rigetti-qiskit.ipynb

```diff
@@ -0,0 +1,380 @@
+        "\n",
+        "| Target name | Target ID | Number of qubits | Description |\n",
+        "| --- | --- | --- | --- |\n",
+        "| Rigetti QVM (simulator) | `rigetti.sim.qvm` | ? qubits | Rigetti's cloud-based \"Quantum Virtual Machine\" simulator. Can run with or without noise. |\n",
+        "| Aspen 11 (hardware) | `rigetti.qpu.aspen-11` | 40 qubits | Rigetti's \"Aspen-11\" universal, physical QPU. Read more [here](https://qcs.rigetti.com/qpus). |\n",
+        "| Aspen M-1 (hardware) | `rigetti.qpu.aspen-m-1` | 80 qubits | Rigetti's \"Aspen-M-1\" universal, physical QPU. Read more [here](https://qcs.rigetti.com/qpus) |\n",
+        "\n",
+        "For this example, we will use `rigetti.sim.qvm`. To learn more about Rigetti's targets, check out our [documentation](https://docs.microsoft.com/azure/quantum/provider-rigetti)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "1b094a8a-76f0-4a49-a0b6-3736f73c18a8",
+      "metadata": {},
+      "source": [
+        "### 2. Build the quantum program\n",
+        "\n",
+        "Let's create a simple Qiskit circuit to run."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 45,
+      "id": "7b42a2c4-cf76-4678-bdce-54ba6d6679e6",
+      "metadata": {},
+      "outputs": [
+        {
+          "data": {
+            "text/html": [
+              "<pre style=\"word-wrap: normal;white-space: pre;background: #fff0;line-height: 1.1;font-family: &quot;Courier New&quot;,Courier,monospace\">     ┌───┐┌─┐\n",
+              "  q: ┤ H ├┤M├\n",
+              "     └───┘└╥┘\n",
+              "c: 1/══════╩═\n",
+              "           0 </pre>"
+            ],
+            "text/plain": [
+              "     ┌───┐┌─┐\n",
+              "  q: ┤ H ├┤M├\n",
+              "     └───┘└╥┘\n",
+              "c: 1/══════╩═\n",
+              "           0 "
+            ]
+          },
+          "execution_count": 45,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "# Create a quantum circuit acting on a single qubit\n",
+        "circuit = QuantumCircuit(1,1)\n",
+        "circuit.name = \"Single qubit random\"\n",
+        "circuit.h(0)\n",
+        "circuit.measure(0, 0)\n",
+        "\n",
+        "# Print out the circuit\n",
+        "circuit.draw()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "23f20a40-c972-470d-9c0e-5cd6c1c2eb38",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "The circuit you built is a simple quantum random bit generator. With Rigetti's simulator, we will be able to simulate the probability of measuring a `1` or `0`."
```

Comment:
Does Rigetti's simulator return counterfactual probabilities (`shots = ∞`), or does it estimate by simulating the given number of shots?

Reply:
Seems that it estimates given shot count. That's why I said that we can _simulate_ the probabilities. Should I clarify?

Reply:
I suppose I should have said "estimate" instead of "simulate" - will fix.